salmon/tagger/cover.py

```diff
@@ -9,12 +9,26 @@
 
 
 def download_cover_if_nonexistent(path, cover_url):
+    """
+    source folder path, url for cover image to be downloaded from
+    returns local source path of cover image, and whether image was downloaded
+    """
+    # use local file if matches filter
     for filename in os.listdir(path):
         if re.match(r"^(cover|folder)\.(jpe?g|png)$", filename, flags=re.IGNORECASE):
-            return
+            cover_path = os.path.join(path, filename)
+            click.secho(f"\nUsing existing cover image found: {filename}...", fg="yellow")
+            return cover_path, False
+    # use url provided and return the image was downloaded
     if cover_url:
-        click.secho("\nDownloading cover image...", fg="yellow")
-        _download_cover(path, cover_url)
+        click.secho("\nDownloading Cover Image...", fg="yellow")
+        cover_path = _download_cover(path, cover_url)
+        if cover_path:
+            return cover_path, True
+    click.secho(
+        "\nNo appropriately named Cover Image in Source Folder", fg="red"
+    )
+    return None
```

Comment:
```suggestion
    return None, None
```
To stay coherent with the other return values of this function

Reply:
Good catch!